pkg/kubelet/kubelet.go

```diff
@@ -697,8 +702,9 @@ func makeBinds(container *api.Container, podVolumes volumeMap) (binds []string)
 func (kl *Kubelet) GenerateRunContainerOptions(pod *api.Pod, container *api.Container, netMode, ipcMode string) (*kubecontainer.RunContainerOptions, error) {
 	var err error
 	opts := &kubecontainer.RunContainerOptions{
-		NetMode: netMode,
-		IpcMode: ipcMode,
+		NetMode:    netMode,
+		IpcMode:    ipcMode,
+		CgroupRoot: kl.cgroupRoot,
```

Comment:
@thockin I think the pod-level cgroup should be added in a separate PR, and probably post-1.0 since it needs some discussion and is orthogonal to this change. In its current form the PR is a minimal change that we need for k8s on Mesos and is backward (and forward) compatible.

The only other question is, should the kubelet resource container live under the cgroup root as well? I'd say yes, to help with resource accounting. So the hierarchy could look like:

```
/kubernetes/kubelet
/kubernetes/pods/pod-uid/container-id
...
```

If there is agreement I can make that change.